src/components/PushChat/PushChatComponent.tsx

```diff
@@ -160,6 +162,7 @@ const ButtonItem = styled(Button)`
 const Header = styled.h3`
   font-size: 46px;
   font-weight: 400;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family